ui/src/app/cohort-review/cohort-review.module.ts

```diff
@@ -1,37 +1,55 @@
-/* tslint:disable:max-line-length */
 import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
 import {ReactiveFormsModule} from '@angular/forms';
 import {ClarityModule} from '@clr/angular';
 
 import {ChartsModule} from '../charts/charts.module';
 
-import {AnnotationItemComponent} from './annotation-item/annotation-item.component';
-import {AnnotationListComponent} from './annotation-list/annotation-list.component';
-import {SetAnnotationCreateComponent} from './set-annotation-create/set-annotation-create.component';
-import {SetAnnotationItemComponent} from './set-annotation-item/set-annotation-item.component';
-import {SetAnnotationListComponent} from './set-annotation-list/set-annotation-list.component';
-import {SetAnnotationModalComponent} from './set-annotation-modal/set-annotation-modal.component';
+import {AnnotationItemComponent
```

Comment:
Is this change related to the PR? Was this just to trigger the CI check? The previous file should have passed codacy as well, right?

Reply:
Unrelated to the PR; just to trigger Codacy.  It looks like Codacy only pays attention to diffs, so I had to generate a diff.  Only `.codacy.yml` would be checked in with this PR; the other file would be reverted.

Reply:
Its a bit clearer if you look at the diffs for each commit separately - the first commit triggered no change in Codacy; after the second commit Codacy had declared that all the `' should be "` "issues" _just for the changed lines_ had been "fixed."  In actuality, they were never issues, but I guess Codacy doesn't have any way of knowing that, Codacy just knows that this line used to be a problem and now its not.

Honestly, given that we're already using automated linters, we could probably just turn off the "stylistic issues" section of codacy altogether and not lose out on anything, unless that would uncheck a box for security freeze.